src/GeocodeSharp/Google/GeocodeClient.cs

```diff
@@ -28,7 +28,7 @@ public GeocodeClient()
         public GeocodeClient(string apiKey)
         {
             _apiKey = apiKey;
-            _baseUrl = string.Format("https://maps.googleapis.com/maps/api/geocode/json?key={0}", Uri.EscapeDataString(_apiKey));
+            _baseUrl = string.Format("https://maps.googleapis.com/maps/api/geocode/json?key={0}&", Uri.EscapeDataString(_apiKey));
```

Comment:
Actually I merged this too quickly. This doesn't fix anything. Did you have a problem initially?